File: app/src/main/java/workshop1024/com/xproject/controller/activity/XActivity.java
package workshop1024.com.xproject.controller.activity;

import android.support.v7.app.AppCompatActivity;

public class XActivity extends AppCompatActivity {
    boolean mIsForeground;

    @Override
    protected void onResume() {
        super.onResume();
        mIsForeground = true;
    }

    @Override
    protected void onPause() {
        super.onPause();
        mIsForeground = false;
    }
}

File: app/src/main/java/workshop1024/com/xproject/controller/activity/XActivity.kt
package workshop1024.com.xproject.controller.activity

import android.support.v7.app.AppCompatActivity

open class XActivity : AppCompatActivity() {
    var mIsForeground: Boolean = false

    override fun onResume() {
        super.onResume()
        mIsForeground = true
    }

    override fun onPause() {
        super.onPause()
        mIsForeground = false
    }
}
